### PR TITLE
Jason/issue 406

### PR DESCRIPTION
**Pull request by duidae**
issue: https://app.asana.com/0/1202896150264815/1202959962459829
companion PR: https://github.com/twreporter/plate/pull/336

### CODE REVIEW

File: admin/client/views/latestList.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+const latestColumnTextStyle = {
+  color: '#999',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis'
+};
+
+const latestColumns = [
+  {
+    name: '最新',
```
Comment (Aylie-Chou) [inline admin/client/views/latestList.js:37]: Based on design doc, the wording should be `標籤名稱`.
螢幕截圖 2023-01-18 下午4 44 15

File: admin/client/views/latestList.js [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+      </InputGroup.Section>
+    );
+  },
+  renderHeader() {
+    let { list } = this.state;
+    return (
+      <div className="ListHeader">
+        <Container>
+          <h2 className="ListHeader__title">
+            {`${plural(this.state.latests.length, ('* ' + list.singular), ('* ' + list.plural))} sorted by 最新`}
```
Comment (Aylie-Chou) [inline admin/client/views/latestList.js:223]: The wording of sorting should be `顯示順序`.
螢幕截圖 2023-01-18 下午4 45 34

**Comment by Aylie-Chou** [comment]
Question: would we implement `save` button in this PR ?

File: admin/client/components/CreateLatestModal.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    return {
+      err: null,
+      values: {},
+      isOpen: false,
+    };
+  },
+  getInitialState() {
+    var values = Object.assign({}, this.props.values);
+
+    Object.keys(this.props.list.fields).forEach(key => {
```
Comment (Aylie-Chou) [inline admin/client/components/CreateLatestModal.js:27]: Could there be other problem if we simplify this function to following ?
```
this.props.list.fields.forEach(field => {
  if (!values[field.path]) ...
})

File: admin/client/components/CreateLatestModal.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+      var nameFieldProps = this.getFieldProps(nameField);
+      nameFieldProps.ref = focusRef = 'focusTarget';
+      if (nameField.type === 'text') {
+        nameFieldProps.className = 'item-name-field';
+        nameFieldProps.placeholder = nameField.label;
+        nameFieldProps.label = false;
+      }
+    }
+
+    Object.keys(list.initialFields).forEach(key => {
```
Comment (Aylie-Chou) [inline admin/client/components/CreateLatestModal.js:141]: Same as line27, could we simplify this funtion to:
```
list.initialFields.forEach(initialField => {
  var field = list.fields[initialField]
  ...
})

File: admin/client/views/home.js [hunk 5]
```diff
@@ -19,16 +21,17 @@ var ListTile = React.createClass({
     count: React.PropTypes.string,
     href: React.PropTypes.string,
     label: React.PropTypes.string,
+    noEdit: React.PropTypes.bool,
```
Comment (Aylie-Chou) [inline admin/client/views/home.js:24]: Question:
What is the purpose of `noEdit` props ?

**Comment by Aylie-Chou** [comment]
Question:
I notice when typing something on add tag window, only one tag would be added in a time, meanwhile if we choose tags without typing, mutiple tags could be added.
Is that the feature or bug ? 😂

https://user-images.githubusercontent.com/48147686/213254918-8ab58de3-35ba-4ee6-adc4-854b38fa2ef5.mov

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Great work !
Just one stand of `saving` button to the goal.
I add some comments about simplifing code & some queations, please have a look.

File: admin/client/components/CreateLatestModal.js (hunk 4 above)
Comment (duidae) [reply admin/client/components/CreateLatestModal.js]: removed.

File: admin/client/views/home.js (hunk 5 above)
Comment (duidae) [reply admin/client/views/home.js:24]: Screen Shot 2023-01-20 at 10 21 06 PM
`noEdit` prop is to disable the `add` button in latest tile

**Comment by duidae** [comment]
> Question: I notice when typing something on add tag window, only one tag would be added in a time, meanwhile if we choose tags without typing, mutiple tags could be added. Is that the feature or bug ? 😂
> 
>  2023-01-19.1.36.54.mov

it's a bug, fixed.

**Comment by duidae** [comment]
> Great work ! Just one stand of `saving` button to the goal. I add some comments about simplifing code & some queations, please have a look.

after some thoughts, add save button for providing `edit state` for users.
please have a look, thanks.

**Review by Aylie-Chou — APPROVED** [review]
LGTM 👍

**Pull request merged**